### PR TITLE
fix: text for revoke flow

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- Updated the text as discussed in the story review meeting.
- from 'file' to 'document' 
- remove '(s)' for revoke flow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -136,4 +136,4 @@
     "typescript": "^4.3.2",
     "wait-on": "^6.0.0"
   }
-}
\ No newline at end of file
+}
```
Comment (simboonlong) [inline package.json]: omit no change files, same with package lock

Comment (isaackps) [reply package.json]: ok but shouldn't we have a empty line at the bottom?

Comment (simboonlong) [reply package.json]: depends on eslint rule, if thats the case ok lo. probably this file was b4 the rule kick in, not sure

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**